isis/src/mro/apps/hideal2pds/main.cpp

```diff
@@ -30,6 +31,7 @@ using namespace Isis;
 using namespace std;
 
 double *g_min, *g_max;
+PixelType g_oType;
```

Comment:
This shouldn't be a global variable. It is only used in the main and can be local.

Reply:
Actually this variable can just be removed. This variable is set but never actually used except for being explicitly set and then passed to a function. The value that this variable was being set to can just be passed to the function.